src/main/java/emissary/place/KffHashPlace.java

```diff
@@ -58,7 +59,11 @@ public void process(IBaseDataObject payload) throws ResourceException {
             return;
         }
 
-        kff.hash(payload);
+        try {
+            kff.hash(payload, true);
```

Comment:
The hard-coded value `true` here dictates whether the KFF hashing prefers the payload's `SeekableByteChannelFactory` vs its `data` byte array.  Could we modify this so that the value passed here is driven by a configuration property rather than hard-coded?  For the foreseeable future, the _vast_ majority of BDOs will have a populated byte array and no SCBF, an it would be nice if we could toggle the preferred option without having to rebuild Emisssary and then reintegrate.

Reply:
Sure - added as a new commit to this PR, let me know if you want any further tweaks with it.